pyrsistent/_checked_types.py

```diff
@@ -208,19 +242,19 @@ def optional(*typs):
 
 
 def _checked_type_create(cls, source_data, _factory_fields=None):
-        if isinstance(source_data, cls):
```

Comment:
no idea what was going on here 🤷

Reply:
You mean the extra indent? Probably a mistake, good that you fixed it.